content/en/docs/collector/deployment/best-practices.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Best Practices
+description: How to configure collectors for various use cases
+weight: 4
+---
+
+Now that you are equipped with the essential deployment patterns for the
+collector, let's have a closer look at best practices for collector (pipeline)
+configurations for different use cases.
+
+## Fan Out
+
+Export signals into more than one backend. For example, for testing, policy
+(signals from dev/test go into backend X, prod goes into Y), or migrations from
+one back-end to another (cut-over) or different signal types go into different
+backends.
+
+<!-- prettier-ignore-start -->
+{{< ot-tabs "Per type" "Per env" "Migration" >}}
+{{< ot-tab lang="yaml">}}
+receivers:
+  otlp:
+    protocols:
+      grpc:
+
+exporters:
+  jaeger:
+    endpoint: "https://jaeger.example.com:14250"
+    insecure: true
```

Comment:
We have [deprecated the Jaeger exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/jaegerexporter#deprecated-jaeger-grpc-exporter), so I would recommend using the OTLP exporter instead